src/modes/static.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(ctx) {
+  return {
+    stop: function() {
+      ctx.map.doubleClickZoom.enable();
```

Comment:
Yup, I think this is mostly me being sloppy when I was first adding it for myself. I'll remove.